jujugui/static/gui/src/app/app.js

```diff
@@ -2634,6 +2636,36 @@ YUI.add('juju-gui', function(Y) {
       maasContainer.show();
     },
 
+    // TODO: Move to views/utils.js.
+    showSharing: function(show = true) {
```

Comment:
Yeah, I was supposed to make that change before making the PR. Will fix and add a docstring.